.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Scope 
+Description of feature / issue, and links to issue...
+
+### Work Done
+ What you did, and where you did it...
```

Comment:
Also think this default text should be removed since it will be removed every.single.time. and will become annoying.